components/mdx/Code.tsx

```
@@ -21,64 +21,109 @@ export function InlineCode({ children }: { children: React.ReactNode }) {
+      {showCopy && (
+        <div
+          className="absolute right-2 top-2 opacity-0 group-hover:opacity-100 transition-opacity"
+          style={{ zIndex: 1 }}
+        >
+          <CopyButton text={codeText} />
+        </div>
+      )}
       {highlightedHtml ? (
         <div
-          // Shiki returns a full <pre class="shiki">...</pre> block
           dangerouslySetInnerHTML={{ __html: highlightedHtml }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)